spec/support/monitor/monitor_idv_steps.rb

```diff
@@ -42,10 +42,10 @@ def verify_identity_with_doc_auth
     expect(page).to have_current_path('/verify/doc_auth/verify')
 
     click_on 'Continue'
-    expect(page).to have_current_path('/verify/phone')
+    sleep 1 while page.has_content?('This might take up to a minute')
 
     click_on 'Continue'
-    expect(page).to have_current_path('/verify/review')
+    expect(page).to have_current_path('/verify/doc_auth/verify')
```

Comment:
I've been reading up a bit on `has_current_path`, and it seems like it should have some built-in retry or waiting mechanics that can give a bit of tolerance to delays? Is it merely the case the flakiness comes from exceeding that maximum allowable wait time? If so, would it be enough to pass a longer [`wait` option](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSessionMatchers:has_current_path%3F) here?

```suggestion
    expect(page).to have_current_path('/verify/doc_auth/verify', wait: 10)
```

There's also `using_wait_time`, though I assume it should be effectively the same:

```suggestion
    using_wait_time(10) do
      expect(page).to have_current_path('/verify/doc_auth/verify')
    end
```

Reply:
Oh nice, I wasn't aware of these. Thanks! The bummer is the wait time is more like 30s, but I'll give these a shot

Reply:
Updated in 09daf85